sections/03_categorize.md

```diff
@@ -19,7 +19,7 @@ tackle any of them? Are there example approaches whereby deep learning is
 already having a transformational impact? I (Casey) have added some sections
 below where I think we could contribute to the field with our discussion.*
 
-### Major challenges
+### Major Areas of Existing Contributions
```

Comment:
Should we settle on title case or lower case for section and sub-section headings?